enterprise/api_client/enterprise_catalog.py

```diff
@@ -6,21 +6,20 @@
 from collections import OrderedDict
 from logging import getLogger
 
-from edx_rest_api_client.client import EdxRestApiClient
-from requests.exceptions import ConnectionError, Timeout  # pylint: disable=redefined-builtin
-from slumber.exceptions import HttpNotFoundError, SlumberBaseException
+from requests.exceptions import ConnectionError, RequestException, Timeout  # pylint: disable=redefined-builtin
 
 from django.conf import settings
 
 from enterprise import utils
-from enterprise.api_client.lms import JwtLmsApiClient
+from enterprise.api_client.client import NoAuthAPIClient, UserAPIClient
+
 
 LOGGER = getLogger(__name__)
 
 
-class EnterpriseCatalogApiClient(JwtLmsApiClient):
+class EnterpriseCatalogApiClient(UserAPIClient):
     """
-    Object builds an API client to make calls to the Enterprise Catalog API.
+    The API client to make calls to the Enterprise Catalog API.
     """
 
     API_BASE_URL = settings.ENTERPRISE_CATALOG_INTERNAL_ROOT_URL + '/api/v1/'
```

Comment:
should we use urljoin here? is there a trailing slash in settings. ENTERPRISE_CATALOG_INTERNAL_ROOT_URL?

Reply:
yeap, we can use urljoin here, thanks
fixed everywhere